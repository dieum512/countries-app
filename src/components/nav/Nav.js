import { Link } from 'react-router-dom';
import backArrow from '../images/backArrow.png'
import settingsIcon from '../images/setting.png'
import micIcon from '../images/mic.png'
import './nav.css'
    
const Nav = () => {
    return (
        <>
            <nav>
                <Link to="/"><img src={backArrow} alt="back-arrow"/></Link>
                <h1>countries</h1>
                <ul className="ul-nav">
                    <li><a href="#"><img src={micIcon} alt="microphone"/></a></li>
                    <li><a href="#"><img src={settingsIcon} alt="settings"/></a></li>
                </ul>
            </nav>
        </>
    )
};

export default Nav;