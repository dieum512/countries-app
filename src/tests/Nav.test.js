import React from 'react';
import renderer from 'react-test-renderer';
import { BrowserRouter as Router } from 'react-router-dom';
import Nav from '../components/nav/Nav';

describe('NavBar', () => {
  it('Should renders correctly', () => {
    const component = renderer.create(
      <Router>
        <Nav />
      </Router>,
    );
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
