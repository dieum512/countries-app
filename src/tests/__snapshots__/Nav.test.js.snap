// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Nav Should renders correctly 1`] = `
<nav
  className="nav-bar"
>
  <a
    href="/"
    onClick={[Function]}
  >
    <img
      alt="back-arrow"
      src="backArrow.png"
    />
  </a>
  <h1>
    Countries
  </h1>
  <ul
    className="ul-nav"
  >
    <li>
      <button 
        type="button"
      >
        <img 
          src=mic.png 
          alt="microphone" 
        />
      </button>
    </li>
    <li>
      <button 
        type="button"
      >
        <img 
          src=setting.png 
          alt="settings" 
        />
      </button>
    </li>
  </ul>
</nav>
`;
